### ISSUE TITLE
all Hotkeys should speak in Microsoft-Word

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#11038]
In Microsoft-word, CTRL+m, CTRL+q, CTRL+s, CTRL+t, CTRL+W, CTRL+F1 and other hotkeys are not announced by NVDA. 

### Is your feature request related to a problem? Please describe.
Yes, Users get trouble of knowing the actions they are doing. they keep on assuming that, for instance, if the document is saved, hanged in a paragraph, and the like. if these hotkeys are announced by NVDA, it helps users to easily understand the action he/she is doing.
### Describe the solution you'd like
All hotkeys should be announced by NVDA.
### Describe alternatives you've considered
No Alternative

### Additional context
when writing

**Comment by Adriani90** [comment]
There are already some issues already open, regarding many hot keys.
With UIA in MS Office 365, many of these hot keys are announced. Unfortunately this is not possible to report this in all MS Word versions because the information about the action is not exposed in earlier such as MS Word 2013 or older.

cc: @lukaszgo1 what do you think?

**Comment by lukaszgo1** [comment]
Many of these are covered by https://github.com/nvaccess/nvda/pull/10415 and #10283. Ctrl+s and Ctrl+f1 are not changing text. In the former case I do not think there is any visual indication of the document being saved though I might be wrong here. In case of minimizing/maximizing ribbon with Ctrl+F1 I don not think it deserves to be announced though technically it should be possible.
